extensions/windows-event-log/ConsumeWindowsEventLog.cpp

```diff
@@ -189,11 +279,24 @@ bool ConsumeWindowsEventLog::subscribe(const std::shared_ptr<core::ProcessContex
               }
 
               size = used;
-              std::vector<char> buf(size);
+              std::vector<wchar_t> buf(size/2);
```

Comment:
Is that size always sufficient? 
The division truncates here.

Reply:
Added 1, just in case.